src/Dscp/DB/SQLite/Types.hs

```diff
@@ -0,0 +1,30 @@
+module Dscp.DB.SQLite.Types
+       ( MonadSQLiteCtx
+       , SQLiteDBLocation (..)
+       , SQLiteDB (..)
+       , SQLiteParams (..)
+       ) where
+
+import Universum
+
+import Database.SQLite.Simple (Connection)
+import Ether.Internal (HasLens)
+
+-- | Set of constraints necessary to operate on SQLite database.
+type MonadSQLiteCtx ctx m =
+    ( MonadReader ctx m
+    , HasLens SQLiteDB ctx SQLiteDB
```

Comment:
`HasLens'`

Reply:
I'll replace all occurrences later, currently it seems to be accessible only from your custom lootbox commit